rtengine/procparams.cc

```diff
@@ -129,22 +131,22 @@ RetinexParams::RetinexParams ()
     setDefaults ();
 }
 
-void RetinexParams::getDefaulttransmissionCurve(std::vector<double> &curve)
+void RetinexParams::getDefaulttransmissionCurve (std::vector<double> &curve)
```

Comment:
It could be that it's the code in `dev` which is "incorrectly" styled and @Desmis 's patch is correctly styled. Either way, one way to remove the style changes from the diff is to run those 12 files in `dev` through astyle, then to run the same 12 files in this branch through astyle (I know this has already been done, but it won't hurt to do it again, just to make sure that the person who astyles `dev` is the same person who astyles this branch, and so that the same version of astyle is used on both).